### PR TITLE
add main CLI

### PR DESCRIPTION
**Pull request by CoolPlayLin**

### CODE REVIEW

File: src/revChatGPT/V3.py [hunk 1]
```diff
@@ -12,7 +12,7 @@
 import requests
 import tiktoken
 
-from . import typings as t
```
Comment (acheong08) [inline src/revChatGPT/V3.py:15]: Why remove the `s` lol

File: src/revChatGPT/typing.py
Comment (acheong08) [inline src/revChatGPT/typing.py:1]: And why rename?

**Review by acheong08 — CHANGES_REQUESTED** [review]
Minor preferences in plurality

File: src/revChatGPT/V3.py (hunk 1 above)
Comment (CoolPlayLin) [reply src/revChatGPT/V3.py:15]: rollback

**Review by acheong08 — APPROVED** [review]

**Pull request merged**